pkg/roachprod/vm/aws/config.go

```diff
@@ -175,7 +176,7 @@ func (c *awsConfigValue) Set(path string) (err error) {
 		return err
 	}
 	// Update the provider's config with the user-specified config.
-	providerInstance.Config = &c.awsConfig
```

Comment:
Looks like `providerInstance` is now redundant, but it's still referenced in `ConfigureClusterCleanupFlags` and `ConfigureCreateFlags`. Mind refactoring so that we can drop `providerInstance`?